Automattic/wp-calypso
# A4A: improve referral dashboard cards layout

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/1579

## Proposed Changes

This PR refactors Referral Dashboard Cards UI to be more responsive and aligned with designs: fuufP6VNfZXYmvLsTRWRlG-fi-8332_118153

If you notice better ways to organise styles please let me know.

Note: tooltips, info are not yet consistent and will be addressed in a separate task.

| Desktop | Mobile |
| --- | --- |
| <img width="1344" alt="Screenshot 2024-12-16 at 10 12 09 AM" src="https://github.com/user-attachments/assets/07348424-29a2-4b99-ba18-9387ab6ea35e" /> | <img width="295" alt="Screenshot 2024-12-16 at 1 35 45 PM" src="https://github.com/user-attachments/assets/1574fd16-9bf5-4135-8677-1b6f195d65ed" /> |
| <img width="1359" alt="Screenshot 2024-12-16 at 1 58 30 PM" src="https://github.com/user-attachments/assets/f20c73c0-5ade-480b-8a7a-15f9248a3575" /> | <img width="387" alt="Screenshot 2024-12-16 at 1 58 22 PM" src="https://github.com/user-attachments/assets/47fd6bf3-ab04-4756-9c43-eba5c81dd438" /> |

## Why are these changes being made?



## Testing Instructions



- Using live link below navigate to `/referrals/dashboard` and check cards responsiveness and compare with designs. 

## Pre-merge Checklist
